assets/js/components/data-table.js

```diff
@@ -91,13 +92,17 @@ export const getDataTableFromData = ( data, headers, options ) => {
 	} );
 
 	return (
-		<div className={ `googlesitekit-table${ ( options && options.disableListMode ? '' : ' googlesitekit-table--with-list' ) }` }>
-			<table
-				className={ `
-					googlesitekit-table__wrapper
-					googlesitekit-table__wrapper--${ data && data[ 0 ] ? data[ 0 ].length : 1 }-col
-				` }
-			>
+		<div className={ classnames(
+			'googlesitekit-table',
+			{ 'googlesitekit-table--with-list': ! options || ! options.disableListMode }
+		) }>
+			<table className={ classnames(
+				'googlesitekit-table__wrapper',
+				{
+					'googlesitekit-table__wrapper--1-col': ! data || ! data[ 0 ] || ! data[ 0 ].length,
+					[ `googlesitekit-table__wrapper--${ data[ 0 ].length }-col` ]: data && data[ 0 ] && data[ 0 ].length,
```

Comment:
Couldn't this be problematic? I'm not entirely sure, but I would think this object property gets evaluated regardless, which would be problematic if e.g. `! data`.

Reply:
Huh, that's a good catch! (I really wish this had a test for `false`-y `data` values. But that's true; it will still get evaluated and `TypeError`.

I'd say it's best to refactor this component to have a `column` value that's used here instead, eg:

```js
const columns = data && data[ 0 ] && data[ 0 ].length ? data[ 0 ].length : 1;

[...]
<table className={ classnames(
  'googlesitekit-table__wrapper',
  `googlesitekit-table__wrapper--${ columns }-col`
) }>
```

Good catch 👍